src/redux/actions/profile.js

```diff
@@ -0,0 +1,36 @@
+import { api } from '../../utils/api';
+
+import {
+  EDIT_PROFILE,
+  PROFILE_FETCH,
+  UPDATE_IMAGE,
+} from '../../constants';
+
+export const profileFetch = payload => ({
+  type: PROFILE_FETCH,
+  payload,
+});
+
+export const onEditProfile = payload => ({
+  type: EDIT_PROFILE,
+  payload,
+});
+
+export const updateImage = payload => ({
+  type: UPDATE_IMAGE,
+  payload,
+});
+
+
+export const getProfile = username => dispatch => api.user.profile(username)
+  .then((response) => {
+    dispatch(profileFetch(response.data));
```

Comment:
this action is dispatched after a profile has been fetched, you might consider renaming it to be more descriptive. `profileFetched` maybe

Reply:
Okay